File: 9041 - Software Instruction/lab07/count_word.py
#!/usr/bin/python

import re,sys

count = 0
key = sys.argv[1]

for line in sys.stdin:
	line = line.lower()
	words = re.split(r"[^a-zA-Z]", line)
	for word in words:
		if word == key:
			count += 1;


print (key + " occurred " + str(count) + " times")

File: 9041 - Software Instruction/lab07/count_word.pl
#!/usr/bin/perl -w

@input = <STDIN>;
$word = $ARGV[0];
$times = 0;

sub countStr
{
	$count = 0;
	foreach $str (@_)
	{
		chomp $str;
		$str=lc($str);
		if ($str eq $word)
		{
			$count++;
		}
	}
	return $count;	
}

foreach $line (@input)
{
	chomp $line;
	@newLine = split(/[^a-zA-Z]+/, $line);	
	$times += countStr(@newLine);
}

print "$word occurred $times times\n";

